packages/cli/src/server/http-server.ts

```diff
@@ -1,34 +1,33 @@
 import * as http from 'http';
 import * as fs from 'fs';
 import * as path from 'path';
-import { SERVER_HOST, SERVER_PORT, REDIRECT_ROUTE, WIDGET_DEMO_ROUTH } from '../constants';
+import { SERVER_HOST, REDIRECT_ROUTE, WIDGET_DEMO_ROUTH, setAvailablePort, getServerPort } from '../constants';
 import { ConfigService } from '../services';
 
 export class HttpServer {
   private server: http.Server;
   public token: string;
   private config: ConfigService = new ConfigService();
 
-  public listen(): Promise<void> {
-    return new Promise((resolve) => {
```

Comment:
Did a small refactor here because i've found that it's not really helpful or needed to wrap this with a new Promise